metabase/metabase
# Convert `calendar.module.css` to CSS Modules

Closes https://github.com/metabase/metabase/issues/40349

### Description

Converts `calendar.module.css` to CSS modules. This file is a local module that only modifies the Calendar component that's used in the Events modal.

### How to verify

Go to a collection and click on `Events` (the calendar icon in the top right of the collections browser). Add an event.

<img width="695" alt="image" src="https://github.com/metabase/metabase/assets/25306947/ab0f4d40-ff89-4060-ad6b-8fa912c52025">

See that the date picker still looks the same.
